cypress/integration/vesselCommunications_spec.ts

```diff
@@ -0,0 +1,71 @@
+describe("As a beacon owner, I want to register my communication details so SAR can contact me in an emergency", () => {
```

Comment:
We want to be consistent with our naming of test files for cypress. I would suggest updating the current test named `serviceStartPage.ts` to either `serviceStartPage.spec.ts` or `serviceStartPage_spec.ts`.

According to the [docs](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Test-files), it looks like Cypress will run everything under the `cypress/integration` folder.  In the docs they use `.spec.ts` rather than `_spec.ts` but it doesn't look like it makes a difference

Reply:
Agreed, I've gone with `.spec.ts` 👍